### ISSUE TITLE
Slow version check response

### ISSUE THREAD

**Issue by mikronac** [elastic/logstash#5592]
Checking logstash version consume a lot of CPU and take up to 30 seconds.

root@euirpwhdevlost001:/home/ubuntu# time /opt/logstash/bin/logstash --version
logstash 2.3.1

real	0m11.275s
user	0m9.780s
sys	0m0.252s

- Version: 2.3.1
- Operating System: Ubuntu 14.04

**Comment by jsvd** [comment]
[for 5.x](https://github.com/elastic/logstash/pull/5490) this will be pretty fast:

```
/tmp/logstash-5.0.0 % time bin/logstash -V
logstash 5.0.0
bin/logstash -V  0.01s user 0.02s system 101% cpu 0.029 total
```

**Comment by suyograo** [comment]
This has been fixed in 5.0.
